_data/simulations/moose_psu_1d_IA

```diff
@@ -0,0 +1,58 @@
+---
+  benchmark:
+    id: 1d
+    version: 1
+
+  metadata:
+    summary: Metadata for the Benchmark1 problems
+    timestamp: 02/20/2017
+    author: PC. Simon
+    email: pjs5523@psu.edu
+    timestamp: "Tue Jan 31 21:01:55 2017 -EST"
+    software:
+      name: moose
+      version: git commit 2593a61 on 2017-01-18
+    hardware:
+      computing plateform: Red Hat Entreprise Linux 6 (Cluster)
+      cores: 40
+      details:
+        - name: clock rate
+          values:
+            rate:
+            unit: GHz
+        - name: processor
+          values: ""
+    implementation:
+      repo:
+        url:
+        version:
+      end_condition: Time limit (3000000)
+      details:
+        - name: time_stepper
+          values: IA
+
+  data:
+    - name: run_time
+      values:
+        [
+          {
+            "time": 78280,
+          }
+        ]
+      transform:
+        - type: formula
+          field: value
+          expr: datum.value_m * 1024.0
+    - name: free_energy
+      url: https://gist.githubusercontent.com/simopier/739e1e4cb6e3682badee62dbb242fd98/raw/ec506181dfd410da88ca3760fd800fe930e48338/moose_psu_1d_IA.csv
+      format:
+        type: csv
+        parse:
+          f_density: number
+          time: number
+      transform:
+        - type: formula
+          field: f_density
```

Comment:
This field needs to be `free_energy` not `f_density`. Unfortunately, the linter doesn't check that currently.